### PR TITLE
ASAP-531 Edit Manuscript

### PR DESCRIPTION
**Pull request by gabiayako**

### CODE REVIEW

File: packages/react-components/src/organisms/__tests__/ManuscriptVersionCard.test.tsx [hunk 1]
```diff
@@ -113,33 +134,90 @@ it('displays quick checks when present', () => {
...
+  expect(getAllByText('Connor Commenter')[0]!.closest('a')!.href!).toContain(
+    '/network/users/commenter-id',
   );
-  expect(getAllByText('Team B')[0]!.closest('a')!.href!).toContain(
-    '/network/teams/team-b',
+  expect(getAllByText('Team Commenter')[0]!.closest('a')!.href!).toContain(
+    '/network/teams/team-commenter',
+  );
+});
+it('displays createdBy as fallback for updatedBy when updatedBy is well defined', () => {
```
Comment (AkosuaA) [inline packages/react-components/src/organisms/__tests__/ManuscriptVersionCard.test.tsx:160]:
```suggestion
it('displays createdBy as fallback for updatedBy when updatedBy is not well defined', () => {
```

File: packages/react-components/src/organisms/ManuscriptVersionCard.tsx [hunk 2]
```diff
@@ -401,10 +523,10 @@ const ManuscriptVersionCard: React.FC<ManuscriptVersionCardProps> = ({
               )}
               <Caption accent="lead" noMargin>
                 <div css={userContainerStyles}>
-                  Date added:
+                  Date created:
```
Comment (AkosuaA) [inline packages/react-components/src/organisms/ManuscriptVersionCard.tsx:526]: Just confirming that there was a design change.

**Review by AkosuaA — APPROVED** [review]
Really great work on this! 🎉🎉🎉

File: packages/react-components/src/organisms/ManuscriptVersionCard.tsx (hunk 2 above)
Comment (gabiayako) [reply packages/react-components/src/organisms/ManuscriptVersionCard.tsx:526]: I'm not sure if this was changed across the whole figma, but we discussed this in this thread https://yld.slack.com/archives/CV4UY4411/p1730721284620429?thread_ts=1730460907.784259&cid=CV4UY4411 :)

Comment (AkosuaA) [reply packages/react-components/src/organisms/ManuscriptVersionCard.tsx:526]: Thanks, confirmed

**Pull request merged**